test/runtests.jl

```diff
@@ -1832,6 +1833,67 @@ else  # run HiGHS tests
             @test calculated_ghp_capex ≈ reopt_ghp_capex atol=300
         end
 
+        @testset "Cambium Emissions" begin
+            """
+            1) Location in contiguous US
+                - Correct data from Cambium (returned location and values)
+                - Adjusted for load year vs. Cambium year (which starts on Sunday) vs. AVERT year (2022 currently)
+                - co2 pct increase should be zero
+            2) HI and AK locations
+                - Should use AVERT data and give an "info" message
+                - Adjust for load year vs. AVERT year
+                - co2 pct increase should be the default value unless user provided value 
+            3) International 
+                - all emissions should be zero unless provided
+            """
+            m1 = Model(optimizer_with_attributes(HiGHS.Optimizer, "output_flag" => false, "log_to_console" => false, "mip_rel_gap" => 0.01))
+            m2 = Model(optimizer_with_attributes(HiGHS.Optimizer, "output_flag" => false, "log_to_console" => false, "mip_rel_gap" => 0.01))
+        
+            post_name = "cambium.json" 
+            post = JSON.parsefile("./scenarios/$post_name")
+        
+            cities = Dict(
+                "Denver" => (39.7413753050447, -104.99965032911328),
+                "Fairbanks" => (64.84053664406181, -147.71913656313163),
+                "Santiago" => (-33.44485437650408, -70.69031905547853)
+            )
+        
+            # 1) Location in contiguous US
+            city = "Denver"
+            post["Site"]["latitude"] = cities[city][1]
+            post["Site"]["longitude"] = cities[city][2]
+            post["ElectricLoad"]["loads_kw"] = [20 for i in range(1,8760)]
+            post["ElectricLoad"]["year"] = 2021 # 2021 First day is Fri
+            scen = Scenario(post)
+        
+            @test scen.electric_utility.cambium_emissions_region == "RMPAc"
+            @test sum(scen.electric_utility.emissions_factor_series_lb_CO2_per_kwh) / 8760 ≈ 0.394608 rtol=1e-3
+            @test scen.electric_utility.emissions_factor_series_lb_CO2_per_kwh[1] ≈ 0.677942 rtol=1e-4 # Should start on Friday
+            @test scen.electric_utility.emissions_factor_series_lb_CO2_per_kwh[8760] ≈ 0.6598207198 rtol=1e-5 # Should end on Friday 
+            @test sum(scen.electric_utility.emissions_factor_series_lb_SO2_per_kwh) / 8760 ≈ 0.00061165 rtol=1e-5 # check avg from AVERT data for RM region
```

Comment:
I'm adding a few additional tests here and in the non-CONUS locations for things like the avert and cambium region ID's returned.  Will the  values here change with future defaults, or can we keep legacy files and refer to them in older model years?  I ask only because I wonder if these test values will change if we refresh AVERT numbers.

Reply:
@zolanaj Thank you for adding tests! 

Yes, with this setup many of the test values will change with future updates to default values, for either changes in the Cambium data or AVERT data. I probably should set these tests up to not do that, but may punt on that unless you think it's high priority for this PR!

Reply:
@adfarth Punt away!  The new tests are for things like region detection which should be (more) stable, and I didn't see an easy way of pulling the test value from the data files so I punted on that myself in the review.